README.rst

```diff
@@ -131,33 +136,69 @@ Importing data into Treasure Data in batch manner.
 
    import sys
    import tdclient
-   import time
+   import uuid
    import warnings
 
    if len(sys.argv) <= 1:
        sys.exit(0)
 
    with tdclient.Client() as td:
-       session_name = "session-%d" % (int(time.time()),)
+       session_name = "session-{}".format(uuid.uuid1())
        bulk_import = td.create_bulk_import(session_name, "mydb", "mytbl")
        try:
            for file_name in sys.argv[1:]:
-               part_name = "part-%s" % (file_name,)
+               part_name = "part-{}".format{file_name}
                bulk_import.upload_file(part_name, "json", file_name)
            bulk_import.freeze()
        except:
            bulk_import.delete()
            raise
        bulk_import.perform(wait=True)
        if 0 < bulk_import.error_records:
-           warnings.warn("detected %d error records." % (bulk_import.error_records,))
+           warnings.warn("detected {} error records.".format(bulk_import.error_records))
        if 0 < bulk_import.valid_records:
-           print("imported %d records." % (bulk_import.valid_records,))
+           print("imported {} records.".format(bulk_import.valid_records))
        else:
-           raise(RuntimeError("no records have been imported: %s" % (repr(bulk_import.name),)))
+           raise(RuntimeError("no records have been imported: {}".format(repr(bulk_import.name))))
        bulk_import.commit(wait=True)
        bulk_import.delete()
 
+
+If you want to import data as `msgpack <https://msgpack.org/>`_ format, you can write as follows:
```

Comment:
Nice addition, but it's a bit hard to see diff from the above example. Don't you simply drop the latter lines?:

```py
import io
import time
import uuid
import warning

import tdclient

t1 = int(time.time())
l1 = [{"a": 1, "b": 2, "time": t1}, {"a": 3, "b": 9, "time": t1}]

with tdclient.Client() as td:
    session_name = "session-{}".format(uuid.uuid1())
    bulk_import = td.create_bulk_import(session_name, "mydb", "mytbl")
    try:
        _bytes = tdclient.util.create_msgpack(l1)
        bulk_import.upload_file("part", "msgpack", io.BytesIO(_bytes))
        bulk_import.freeze()
    except:
        bulk_import.delete()
        raise
    bulk_import.perform(wait=True)
    # same as the above example
```

Reply:
sounds good 6fd3a6f